app/src/test/java/com/example/concordia_campus_guide/ModelTests/TestUtils.java

```diff
@@ -0,0 +1,76 @@
+package com.example.concordia_campus_guide.ModelTests;
+
+import com.example.concordia_campus_guide.Models.Building;
+import com.example.concordia_campus_guide.Models.Buildings;
+import com.example.concordia_campus_guide.Models.Coordinates;
+import com.example.concordia_campus_guide.Models.ListOfCoordinates;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class TestUtils {
+
+    public TestUtils(){}
```

Comment:
Yes this is what I meant. However, I was thinking about static fields so that the vars could be called statically without having to instantiate a TestUtils class.

Reply:
Don't forget to add the access modifier keywords for this eh.